certified-connectors/Nodefusion Portal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,166 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Nodefusion Portal",
+    "description": "A Nodefusion Portal Custome Connector is used for various actions, such as opening support case, or subscription activation",

```

Comment:
@hkusulja Please note that this is not a custom connector and it would be a certified connector. Please update your connector description by providing more explanation of your product/service which can be understandable by everyone. 

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
updated